packages/network-of-terms-catalog/catalog/datasets/homosaurus.jsonld

```diff
@@ -0,0 +1,54 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://homosaurus.org/",
```

Comment:
Or should this be https://data.ihlia.nl/homosaurus?

Reply:
I think it should be https://data.ihlia.nl/homosaurus - but I'll verify!